### ISSUE TITLE
Action and video loss flatlines

### ISSUE THREAD

**Issue by safe-sentinel-co** [sharinka0715/X-WAM#3]
We created a simple test using one camera observation plus depth but the action and depth loss seem to flatline. @sharinka0715 any clues why this could be happening? Our actions and state map perfectly to what was used in training

<img width="1208" height="615" alt="Image" src="https://github.com/user-attachments/assets/e6f072ab-b4dc-4407-992a-98ac79b54485" />

**Comment by sharinka0715** [comment]
Hello,
I think the loss is definitely too high. Would you mind providing more details during your training? For instance, batch size, the number of GPUs, learning rate, etc. This would affect the training curve much.

**Comment by safe-sentinel-co** [comment]
I created a repo with all the info:
 https://github.com/safe-sentinel-co/x-wam-umi-finetuning.git

If you prefer, I can post it here as well

**Comment by sharinka0715** [comment]
> I created a repo with all the info: https://github.com/safe-sentinel-co/x-wam-umi-finetuning.git
> 
> If you prefer, I can post it here as well

Thanks for your detailed settings. I think the global batch size (4 GPUs x 8) is too small in your experiments. In fact, we use a global batch size of 128 (32 GPUs  x 4) during fine-tuning the model on RoboCasa and RoboTwin, and the batch size should be much larger when the fine-tuning dataset becomes more complex.

I would recommend you make the changes below:
1. Enable gradient checkpointing to support larger batch size per GPU (add `use_gradient_checkpointing: true` in your config yaml.)
2. Enable gradient accumulation (set `accumulate_grad_batches` larger than 1 in your config yaml) to increase the effective batch size.
3. To increase your training efficiency and focus more on action, you can set `clean_action_ratio: 0.0` to temporarily disable the action-conditioned part.

I hope this can help you.
